### PR TITLE
adds zoom to the examples

### PR DESCRIPTION
**Pull request by MichaelFroelich**

### CODE REVIEW

**Comment by martinfiers** [comment]
Nice! When testing, if you zoom in too much, the tiles just disappear. Perhaps in the example they could be capped so this doesn't happen.

Not really required for this PR, but what if you would want to zoom in even more, do you have any idea how this could be done? (e.g. in the vertex shader use a magnification of, say 8 instead of 4, and start with a transform with a scale of 2).

File: examples/iso_main.rs [hunk 1]
```diff
@@ -45,6 +47,14 @@ fn camera_movement(
 
         let translation = transform.translation();
 
+        if keyboard_input.pressed(KeyCode::Z) {
+            transform.set_scale(scale + 0.1);
+        }
+
+        if keyboard_input.pressed(KeyCode::X) {
+            transform.set_scale(scale - 0.1);
```
Comment (martinfiers) [inline examples/iso_main.rs:55]: Avoid going below 1? (so the tiles don't disappear).

File: examples/iso_main.rs [hunk 2]
```diff
@@ -43,6 +45,14 @@ fn camera_movement(
             direction -= Vec3::new(0.0, 1.0, 0.0);
         }
 
+        if keyboard_input.pressed(KeyCode::Z) {
+            transform.set_scale(scale + 0.1);
+        }
+
+        if keyboard_input.pressed(KeyCode::X)  && scale > 1.1 {
```
Comment (martinfiers) [inline examples/iso_main.rs:52]:
```suggestion
        if keyboard_input.pressed(KeyCode::X)  && scale >= 1.1 {
```

File: examples/ortho_main.rs [hunk 3]
```diff
@@ -43,7 +45,16 @@ fn camera_movement(
             direction -= Vec3::new(0.0, 1.0, 0.0);
         }
 
+        if keyboard_input.pressed(KeyCode::Z) {
+            transform.set_scale(scale + 0.1);
+        }
+
+        if keyboard_input.pressed(KeyCode::X) && scale > 1.1 {
```
Comment (martinfiers) [inline examples/ortho_main.rs:52]:
```suggestion
        if keyboard_input.pressed(KeyCode::X) && scale >= 1.1 {
```

**Review by martinfiers — APPROVED** [review]
Small comment, I think >= is more correct?

**Comment by martinfiers** [comment]
@StarArawn any feedback/can this be merged?

**Comment by StarArawn** [comment]
@martinfiers Sorry, been rather busy as of late! This looks good to me though!

**Pull request merged**